### PR TITLE
install everything needed for pecl

### PR DESCRIPTION
**Pull request by remicollet**
As pecl installation is obviously broken, 
mostly a quickly written PoC.

```
# pecl install -f /work/GIT/pecl-and-ext/ddtrace/datadog_trace-0.43.0.tgz
...

$ DD_TRACE_CLI_ENABLED=1 php -d extension=ddtrace --ri ddtrace

ddtrace


Datadog PHP tracer extension
For help, check out the documentation at https://docs.datadoghq.com/tracing/languages/php/
(c) Datadog 2019

Datadog tracing support => enabled
Version => 1.0.0-nightly

Directive => Local Value => Master Value
ddtrace.disable => Off => Off
ddtrace.internal_blacklisted_modules_list => ionCube Loader,newrelic, => ionCube Loader,newrelic,
ddtrace.request_init_hook => /usr/share/pear/datadog_trace/bridge/dd_wrap_autoloader.php => /usr/share/pear/datadog_trace/bridge/dd_wrap_autoloader.php
ddtrace.strict_mode => Off => Off

$ DD_TRACE_CLI_ENABLED=1 php -d extension=ddtrace src/dd-doctor.php 
WARNING: Script is running from the CLI SAPI.
         Please run this script from your web browser.

DataDog trace extension verification

- PHP version and SAPI                               [7.3.17 - cli]
- ddtrace extension installed                        [OK]
- ddtrace version (installed)                        [1.0.0-nightly]
- ddtrace version (const)                            [1.0.0-nightly]
- ddtrace version (userland)                         [1.0.0-nightly]
- ddtrace versions in sync                           [OK]
- dd_trace() function available                      [OK]
- dd_trace_env_config() function available           [OK]
- ddtrace.request_init_hook set                      [OK]
- ddtrace.request_init_hook reachable                [OK]
- ddtrace.request_init_hook has run                  [OK]
- 'open_basedir' allows access to '/proc/self/'      [OK]
- IntegrationsLoader exists                          [OK]
- Integrations not loaded yet                        [OK]
- Registering an autoloader...
- Integrations loaded                                [OK]
- DDTrace\Tracer class exists                        [OK]
- Background sender is enabled?                      [NO]
  > You can enable the background sender via
    DD_TRACE_BETA_SEND_TRACES_VIA_THREAD=true
- Configured Agent host                              [localhost]
- Configured Agent port                              [8126]
- Agent can receive traces                           [FAIL]
Curl verbose output: 

*   Trying 127.0.0.1:8126...
* TCP_NODELAY set
* connect to 127.0.0.1 port 8126 failed: Connection refused
* Failed to connect to localhost port 8126: Connection refused
* Closing connection 0

```

### CODE REVIEW

File: src/ext/ddtrace.c [hunk 1]
```diff
@@ -46,7 +46,7 @@ STD_PHP_INI_BOOLEAN("ddtrace.disable", "0", PHP_INI_SYSTEM, OnUpdateBool, disabl
                     ddtrace_globals)
 STD_PHP_INI_ENTRY("ddtrace.internal_blacklisted_modules_list", "ionCube Loader,newrelic,", PHP_INI_SYSTEM,
                   OnUpdateString, internal_blacklisted_modules_list, zend_ddtrace_globals, ddtrace_globals)
-STD_PHP_INI_ENTRY("ddtrace.request_init_hook", "", PHP_INI_SYSTEM, OnUpdateString, request_init_hook,
+STD_PHP_INI_ENTRY("ddtrace.request_init_hook", "@php_dir@/datadog_trace/bridge/dd_wrap_autoloader.php", PHP_INI_SYSTEM, OnUpdateString, request_init_hook,
```
Comment (remicollet) [inline src/ext/ddtrace.c:49]: Don't find nice way to make this conditional....

**Comment by morrisonlevi** [comment]
Remi, thanks for opening this PR! I'm not working right now since it's the weekend, but it's a useful PR and wanted to let you know that we are working on compiling all of our sources down to a single file (#840) which will make supporting pecl installs much easier.

There are still some other concerns like the ini file, which I'll look at during the work-week. Have a good weekend!

File: src/ext/ddtrace.c (hunk 1 above)
Comment (remicollet) [reply src/ext/ddtrace.c:50]: found

**Comment by SammyK** [comment]
Thank you so much for this @remicollet!

**Comment by JanThiel** [comment]
Hey @morrisonlevi, just wanted to bump this and ask about the progress with supporting the remi packages for the APM. 
I just noticed that https://github.com/DataDog/dd-trace-php/pull/840 is already merged and public  for around a month now :-). So any chance getting the remi (pecl) package working as well?

Thanks and best regards!

**Comment by SammyK** [comment]
Hey @JanThiel! Thanks for following up. Yes, we definitely have this on our radar and are planning to roll it out in a release soon. I don't many more specifics than that at the moment, but I'll follow up next week to give a better ballpark timeline. :)

**Comment by SammyK** [comment]
@JanThiel Just following up - this will likely be going into 0.48.0 (the release after next). I'll keep you posted as we get closer to merge.

**Pull request merged**